File: fizzBuzz.c
#include <stdio.h>

int main()
{
    int number_1 ;
    printf("Please enter the range : \n");
    scanf("%d", &number_1);
    for( int num = 1; num <= number_1 ; num++)
    {
        
        if (num%3 == 0 && num%5 == 0)
        {
            printf("FizzBuzz \n");
            
        }
        
        else if (num%3 == 0)
        {
            printf("Fizz \n");
            
        }
        else if (num%5 == 0)
        {
            printf("Buzz \n");
            
        }
        else
        {
            printf("%d \n", num);
            
        }
        
    }

    return 0;
    
}

File: FizzBuzz.py
range_1 = int(input("Please enter start number : \n"))
range_2 = int(input("Please enter end number : \n "))
for number in range(range_1,range_2 + 1) :
  if number%3 == 0 and number%5 == 0 :
    print("FizzBuzz")
  elif number%3 == 0 :
    print("Fizz")
  elif number%5 == 0 :
    print("Buzz")
  else:
    print(number)
